### DISCUSSION TITLE
Pyarmor.runtime version not found?

### DISCUSSION THREAD

**Discussion by noahrepublic** [dashingsoft/pyarmor#1158] [Q&A] [answered]
I been trying to get my script to run everywheres, so I switched it to my laptop. When I discovered there was a Pip library for running the scripts with "pip install pyarmor.runtime" however I keep getting this error message: ERROR: Could not find a version that satisfies the requirement pyarmor.runtime (from versions: none)
ERROR: No matching distribution found for pyarmor.runtime

**Comment by jondy** [comment, marked as answer]
Sorry, there is typos in the documentation. It should be `pyarmor.cli.runtime`, try to install this package.

**Reply by noahrepublic** [reply]
![image](https://user-images.githubusercontent.com/75813466/233480165-734960cf-bab2-4419-9ac5-b8579fa7352d.png)

Put when I run the installation it says it is satisfied
